ci/clone_request.py

```diff
@@ -33,24 +35,11 @@ def post(**kwargs):
 
 if __name__ == "__main__":
 
-    # tests s3 hosted 'nteract-notebooks' bucket
-    # res = get("?s3_bucket=nteract-notebooks&s3_key=published/whateverwewant.json")
-    # print(res.content)
-    # res = get("/?s3_bucket=nteract-notebooks&s3_key=published/whateverwewant.json")
-    # print(res.content)
-    # res = get("?s3_bucket=nteract-notebooks&s3_key=Introduction_to_Chainer.ipynb")
-    # print(res.content)
-
-    # tests minio created 'bookstore' bucket
-    # res = get("?s3_bucket=bookstore&s3_key=ci-published/ci-published.ipynb")
-    # print(res.content)
-    # res = post(s3_bucket="nteract-notebooks", s3_key="Introduction_to_Chainer.ipynb")
-    # print(res.content)
-    res = post(s3_bucket="bookstore", s3_key="ci-published/ci-published.ipynb")
-    pprint.pprint(res.json())
-    res = post(
+    response = post(s3_bucket="bookstore", s3_key="ci-published/ci-published.ipynb")
```

Comment:
The comments are purposefully there to illustrate other things that are intended to be easily tested interactively. They should not be removed.

Reply:
Or rather…if we are going to remove these comments, we might as well remove the file in its entirety as it's role is in aiding debugging. I thought including it in the repo would be useful for others to have as an example. 

To be honest, the motivation of the client was to make it easier to do this kind of quick debugging.

Reply:
Honestly, I thought this was uncommented code not a comment for the user of the library. It would be better to perhaps put in a docstring instead and make it clear that these are tips for debugging.

Reply:
If their purpose is that ambiguous shouldn't we just remove them from the library?

Reply:
I've updated the docstrings so that the content remains and is more understandable.